src/components/Navigation/README.md

```diff
@@ -182,6 +182,10 @@ The user menu component displays the current user’s avatar and name, and actio
 | avatarInitials | AvatarProps['initials'] | The merchant’s initials, rendered in place of an avatar image when not provided                         |
 | avatarSource   | AvatarProps['source']   | An avatar image representing the merchant                                                               |
 
+### Deprecation rationale
+
+As of release 3.4.1 `Navigation.UserMenu` will be deprecated in favor of `TopBar.UserMenu`. We're making this change to make room for a soon-to-be-shipped change to the topbar layout in which the `TopBar.UserMenu` will stay visible on mobile.
```

Comment:
Per the deprecation guidelines, `TopBar.UserMenu` should link to that component.

@tmlayton does that mean it should link to the component in github, like this:

```suggestion
As of release 3.4.1 `Navigation.UserMenu` will be deprecated in favor of [`TopBar.UserMenu`](https://github.com/Shopify/polaris-react/blob/master/src/components/TopBar/components/UserMenu/UserMenu.tsx). We're making this change to make room for a soon-to-be-shipped change to the topbar layout in which the `TopBar.UserMenu` will stay visible on mobile.
```

Reply:
If the component has a linkable section in the style guide, then I would add the link. Otherwise, I wouldn’t link to subcomponent code.